install/mysql/glpi-empty.sql

```diff
@@ -4202,21 +4202,126 @@ CREATE TABLE `glpi_monitortypes` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
 
 
-### Dump table glpi_netpoints
+### Dump table glpi_sockets
 
-DROP TABLE IF EXISTS `glpi_netpoints`;
-CREATE TABLE `glpi_netpoints` (
+DROP TABLE IF EXISTS `glpi_sockets`;
+CREATE TABLE `glpi_sockets` (
   `id` int NOT NULL AUTO_INCREMENT,
+  `position` int NOT NULL DEFAULT '0',
   `entities_id` int NOT NULL DEFAULT '0',
+  `is_recursive` tinyint NOT NULL DEFAULT '0',
   `locations_id` int NOT NULL DEFAULT '0',
   `name` varchar(255) DEFAULT NULL,
+  `socketmodels_id` int NOT NULL DEFAULT '0',
+  `wiring_side` tinyint DEFAULT '1',
+  `itemtype` varchar(255) DEFAULT NULL,
+  `items_id` int NOT NULL DEFAULT '0',
+  `networkports_id` int NOT NULL DEFAULT '0',
   `comment` text,
   `date_mod` timestamp NULL DEFAULT NULL,
   `date_creation` timestamp NULL DEFAULT NULL,
   PRIMARY KEY (`id`),
   KEY `name` (`name`),
+  KEY `socketmodels_id` (`socketmodels_id`),
   KEY `complete` (`entities_id`,`locations_id`,`name`),
+  KEY `is_recursive` (`is_recursive`),
   KEY `location_name` (`locations_id`,`name`),
+  KEY `item` (`itemtype`,`items_id`),
+  KEY `networkports_id` (`networkports_id`),
+  KEY `wiring_side` (`wiring_side`),
+  KEY `date_mod` (`date_mod`),
+  KEY `date_creation` (`date_creation`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
+
+
+### Dump table glpi_cables
+
+DROP TABLE IF EXISTS `glpi_cables`;
+CREATE TABLE `glpi_cables` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) DEFAULT NULL,
+  `entities_id` int NOT NULL DEFAULT '0',
+  `is_recursive` tinyint NOT NULL DEFAULT '0',
+  `rear_itemtype` varchar(255) DEFAULT NULL,
+  `front_itemtype` varchar(255) DEFAULT NULL,
+  `rear_items_id` int NOT NULL DEFAULT '0',
+  `front_items_id` int NOT NULL DEFAULT '0',
+  `rear_socketmodels_id` int NOT NULL DEFAULT '0',
+  `front_socketmodels_id` int NOT NULL DEFAULT '0',
+  `rear_sockets_id` int NOT NULL DEFAULT '0',
+  `front_sockets_id` int NOT NULL DEFAULT '0',
```

Comment:
Naming should match cenventions to ensure compatibility with generic logic.

```suggestion
  `itemtype_rear` varchar(255) DEFAULT NULL,
  `itemtype_front` varchar(255) DEFAULT NULL,
  `items_id_rear` int NOT NULL DEFAULT '0',
  `items_id_front` int NOT NULL DEFAULT '0',
  `socketmodels_id_rear` int NOT NULL DEFAULT '0',
  `socketmodels_id_front` int NOT NULL DEFAULT '0',
  `sockets_id_rear` int NOT NULL DEFAULT '0',
  `sockets_id_front` int NOT NULL DEFAULT '0',
```

Reply:
done